src/components/modals/SendFormModal.vue

```diff
@@ -162,10 +162,10 @@ const SendFormModal = defineComponent({
         validators.maxCharacters(128),
       ],
       amount: [
-        0,
+        '',
         validators.required,
         validators.integer,
-        ...validators.num(0),
+        ...validators.num(1, Number(props.balance[0].amount)),
```

Comment:
why you use balance[0]? why if in balance array order will be different? You should use array find or smthng like this, And what if balance will be empty? For example didn't loaded or load failed?